rest/server_context.go

```diff
@@ -946,6 +954,92 @@ func (sc *ServerContext) getDbConfigFromServer(dbName string) (*DbConfig, error)
 	return &config, nil
 }
 
+//////// STATS LOGGING
+
+func (sc *ServerContext) startStatsLogger() {
+
+	statsLogFrequencySecs := 5
```

Comment:
Should define the default value as a constant.

Reply:
Fixed